Prose/ProseLib/Sources/SidebarFeature/Icon.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Icon.swift
+//  Prose
+//
+//  Created by Rémi Bardon on 27/03/2022.
+//
+
+import SwiftUI
+
+enum Icon: String, CaseIterable {
+    case unread = "tray.2"
+    case reply = "arrowshape.turn.up.left.2"
+    case directMessage = "message"
+    case addressBook = "text.book.closed"
+    case group = "circle.grid.2x2"
+
+    var image: Image {
+        Image(systemName: self.rawValue)
+    }
+}
+
+struct Icon_Previews: PreviewProvider {
```

Comment:
Hm, do you think we actually need a preview for a small subset of SF Symbols? This also kind of goes for the actual Enum. Do we define a different Enum in every place where we want to use SF Symbols? What is the benefit in this case since there is virtually no code reuse and the strings have to be hard-coded somewhere?

Reply:
For now, I'm copy-pasting the code I wrote, then I'll translate it to TCA and check that everyting is OK.

Reply:
To solve that problem, I think I'd prefer a snapshot test instead that can be automated. Previews still need to be looked at manually to confirm that everything is working as expected.